js/muzima.js

```diff
@@ -728,6 +728,19 @@ $(document).ready(function () {
 
     /* End - Code to Serialize form along with Data-Concepts */
 
+     document.setupAutoCompleteData = function(elementName) {
+         var dataDictionary = [];
```

Comment:
Hi @nribeka , 
  Its not required to add the variable twice. I will make the changes and generate a new pull request.